packages/x-charts/src/models/axis.ts

```diff
@@ -515,6 +515,45 @@ export interface ChartsAxisData {
   seriesValues: Record<string, number | null | undefined>;
 }
 
+export type CartesianDirection = 'x' | 'y';
+export type PolarDirection = 'rotation' | 'radius';
+
+/**
+ * The data used to identify pointer interaction with axis
+ */
+export interface CartesianAxisItemIdentifier {
+  /**
+   * The axis direction.
+   */
+  direction: CartesianDirection;
+  /**
+   * The axis id.
+   */
+  axisId: AxisId;
```

Comment:
Do we need the direction? The direction can be derived from the list of axes and the axis ID, right? So isn't the direction a bit redundant here?

Reply:
With the current implementation, it would be hard, because 

```
const axis = useXAxis(id) ?? useYAxis(id)
```

On the principle, I agree. The current axis management could be improved to remove this x/y distinction in the storage. But that would be a much larger PR.

Reply:
> But that would be a much larger PR.

Wouldn't it be exporting a `useCartesianAxes` and a `useCartesianAxis` hooks whose implementation is basically `return useXAxis(id) ?? useYAxis(id)`? Or am I missing something?

It would save us from exposing the direction here. 

---

Is there ever a case for a chart with cartesian and polar axes simultaneously? If so, we might have to expose a `useAxes` because a user doesn't know the axis type (cartesian or polar) to use the proper hook.

Reply:
> we still need useXAxis() and useYAxis() because they default to first axis in each direction

Sure, we can keep it, although the difference in behavior should only be theoretical. I believe we issue a warning when an x-axis has the same ID as a y-axis, so in practice the behavior should be the same as users aren't using the same IDs for different axes.